articles/operator-service-manager/helm-requirements.md

```diff
@@ -2,7 +2,7 @@
 title: About Helm package requirements for Azure Operator Service Manager
 description: Learn about the Helm package requirements for  Azure Operator Service Manager.
 author: sherrygonz
-ms.author: sherryg
+ms.author: sushrao
```

Comment:
I don't believe the MS author should be changed. Please revert your change here.